bwa-samtools/0.7.12_1.2.1-254-6462e34/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# Adapted from https://github.com/common-workflow-language/workflows/blob/master/tools/samtools-Dockerfile
+
+FROM dukegcb/bwa:0.7.12
+MAINTAINER Dan Leehr "dan.leehr@duke.edu"
+ENV DEBIAN_FRONTEND noninteractive
+
+WORKDIR /tmp
+
+### Install required packages (samtools)
+
+RUN apt-get clean all &&\
+    apt-get update &&\
+    apt-get install -y  \
+        libncurses5-dev \
+        git && \
+    apt-get clean && \
+    apt-get purge && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*
+
+### Installing samtools/htslib/tabix/bgzip
+
+ENV VERSIONH 1.2.1-254-6462e34
+ENV NAMEH htslib
+ENV URLH "https://github.com/samtools/htslib/archive/${VERSIONH}.tar.gz"
+ENV SHA1H "6462e349d16e83db8647272e4b98d2a92992071f"
```

Comment:
Ah good catch. The part that uses it was commented out in the original. So I deleted the commented-out stuff but didn't catch the unused var. https://github.com/common-workflow-language/workflows/blob/master/tools/samtools-Dockerfile#L54